pom.xml

```diff
@@ -189,6 +193,33 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.pitest</groupId>
+          <artifactId>pitest-maven</artifactId>
+          <version>1.1.11</version>
+          <configuration>
+            <reportsDirectory>local/pit-reports</reportsDirectory>
+            <timestampedReports>false</timestampedReports>
+            <timeoutConstant>1000</timeoutConstant>
+            <targetClasses>
+              <param>${pitest.classes}</param>
```

Comment:
Ah, so by default you'd like to run it without this configuration parameter, but then it's no longer possible to override on the command line?